packages/plugins/crowdloan/src/crowdloan/interfaces.ts

```diff
@@ -87,4 +87,9 @@ export interface Additionals {
     name: String,
     address: AccountId,
     amount: Balance
+}
+
+export interface Removals {
+    name: String,
+    address: AccountId
 }
```

Comment:
```suggestion
}

```

Most editors will add this on save